examples/poetry_example/Dockerfile

```diff
@@ -0,0 +1,21 @@
+##  Could be any base image, only build dependencies are git and pip
+FROM python:3.8-slim
+
+ENV PROJECT_NAME='commands-base-example'
+ARG POETRY_VERSION=1.0.5
+
+ENV PATH="/root/.poetry/bin:${PATH}"
+
+RUN apt-get install -y git && pip3 install --upgrade pip && \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+WORKDIR /the/workdir/path
+ENV ALADDIN_COMMANDS_PATH='/the/workdir/path/commands_base_example'
```

Comment:
I might suggest moving this `ENV` command down to the bottom since that's when we've defined all of the components of the final path (in the `WORKDIR` and `COPY commands_base_example` commands)

Reply:
moved